src/v/kafka/server/partition_proxy.h

```diff
@@ -126,12 +127,7 @@ class partition_proxy {
     std::unique_ptr<impl> _impl;
 };
 
-template<typename Impl, typename... Args>
-partition_proxy make_partition_proxy(Args&&... args) {
-    return partition_proxy(std::make_unique<Impl>(std::forward<Args>(args)...));
-}
-
 std::optional<partition_proxy>
-make_partition_proxy(const model::ntp&, cluster::partition_manager&);
+make_partition_proxy(const model::ktp&, cluster::partition_manager&);
```

Comment:
Arguably this proxy stuff doesn't even have any point now that coproc is removed?

Reply:
>Arguably this proxy stuff doesn't even have any point now that coproc is removed?

correct. we've been slowly removing coproc bits, but some of the proxy stuff has stuck around a bit longer than other things.